docs/configuration/http_filters/fault_filter.rst

```diff
@@ -152,3 +178,5 @@ prefix <config_http_conn_man_stat_prefix>` comes from the owning HTTP connection
 
   delays_injected, Counter, Total requests that were delayed
   aborts_injected, Counter, Total requests that were aborted
+  <downstream-cluster>.delays_injected, Counter, Total delayed requests for the given downstream cluster
```

Comment:
that makes a table (when rendered) expand a lot, i prefer to keep the sentence shorter.